app/services/fastly_vcl/whitelisted_params.rb

```diff
@@ -0,0 +1,58 @@
+module FastlyVCL
+  # Handles updates to our VCL snippet on Fastly that whitelists params
+  class WhitelistedParams
+    VCL_DELIMITER_START = "^(".freeze
+    VCL_DELIMITER_END = ")$".freeze
+    SNIPPET_NAME = "Whitelist certain querystring parameters".freeze
+    FILE_PARAMS = YAML.load_file("config/fastly/whitelisted_params.yml").sort.freeze
+
+    class << self
+      def update
+        fastly = Fastly.new(api_key: ApplicationConfig["FASTLY_API_KEY"])
+        service = fastly.get_service(ApplicationConfig["FASTLY_SERVICE_ID"])
+        latest_version = service.version
+
+        snippet = fastly.get_snippet(ApplicationConfig["FASTLY_SERVICE_ID"],
+                                     latest_version.number,
+                                     SNIPPET_NAME)
+
+        unless params_outdated?(snippet.content)
+          Rails.logger.info("No Fastly VCL updates needed for version #{latest_version.number}.")
```

Comment:
I think this should be 
```ruby 
return unless params_outdated?(snippet.content)
```
and then below I would log WHEN we update since that would probably be more helpful. I would also throw that into Datadog as a metric so if things break anyone can quickly look at Datadog and see that we updated the VCL. I would also add what fields were added or removed, ie the diff, so we can quickly see what changed.

Reply:
Great idea, updated @mstruve. I seriously can't wait to move to Datadog logs. I'm going to rip through the codebase and change a lot of the Datadog calls to log once we do that, including this one 💯